### ISSUE TITLE
Custom $CIRRUS_WORKING_DIR not used

### ISSUE THREAD

**Issue by cevich** [cirruslabs/cirrus-cli#320]
I'm trying to locally test `.cirrus.yml` locally before pushing changes to my repository.  My configuration requires the repository files to be placed into a specific location in the filesystem.  My configuration sets `$CIRRUS_WORKING_DIR` for this, and it works fine in Cirrus-CI, but when run locally it is ignored.

```
/tmp/test $ ls -la
total 4
drwxrwxr-x.  2 cevich cevich   80 Mar  2 10:25 ./
drwxrwxrwt. 59 root   root   2220 Mar  2 10:25 ../
-rw-rw-r--.  1 cevich cevich  198 Mar  2 10:25 .cirrus.yml
-rw-rw-r--.  1 cevich cevich    0 Mar  2 10:17 testfile
/tmp/test $ cat .cirrus.yml
---

env:
    CIRRUS_WORKING_DIR: /usr/src

test_task:
    container:
        image: alpine
    script: |
        /bin/true
        pwd
        ls -la $PWD
        ls -la /tmp/cirrus-ci/working-dir
/tmp/test $ cirrus run --container-backend podman -o simple
Started 'test' task
Started 'Preparing execution environment...'
Preparing volume to work with...
'Preparing execution environment...' succeeded in 2.5s!
Started 'image pull'
Pulling image alpine...
'image pull' succeeded in 10s!
Started 'main' script
/bin/true
pwd
/usr/src
ls -la $PWD
total 0
drwxr-xr-x    2 root     root            42 Mar  2 15:25 .
drwxr-xr-x    8 root     root            17 Mar  2 15:25 ..
ls -la /tmp/cirrus-ci/working-dir
total 4
drwxr-xr-x    2 root     root            41 Mar  2 15:25 .
drwxr-xr-x    3 root     root            48 Mar  2 15:25 ..
-rw-r--r--    1 root     root           198 Mar  2 15:25 .cirrus.yml
-rw-r--r--    1 root     root             0 Mar  2 15:25 testfile

'main' script succeeded in 1.0s!
'test' task succeeded in 14s!
$ cirrus --version
cirrus version 0.37.0-3757693
```

**Comment by fkorotkov** [comment]
Interesting that `pwd` printed the right folder but `$PWD` was set incorrectly. 🤔

**Comment by edigaryev** [comment]
> Interesting that `pwd` printed the right folder but `$PWD` was set incorrectly. 🤔

This happened because the current directory was switched correctly, but it was not populated with the project directory's contents (e.g. `.cirrus.yml`).

#323 fixes this by using separate volumes for agent and project directory's contents, which in turn allows us to mount working volume anywhere.

**Comment by edigaryev** [comment]
Thanks for reporting this!

The fix should be available shortly as a part of [0.38.1 release](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.38.1).

**Comment by cevich** [comment]
Thanks for fixing it :smiley:
